docs/claws-mail.md

```diff
@@ -0,0 +1,72 @@
+# vdirsyncer with Claws Mail
+
+First of all, you must know that you can **not** edit CardDav contacts only **watch** CardDav contacts with Claws-Mail
+
+## Preparation
+First of al you need to install vdirsyncer, for that look [here](https://vdirsyncer.pimutils.org/en/stable/installation.html).
```

Comment:
Use relative links (`installation.html`) rather than absolute ones, so that locally-built documentation is also navigatable (also, in case anything moves in future).